Loader/SymfonyFixturesLoader.php

```diff
@@ -19,6 +19,21 @@
  */
 final class SymfonyFixturesLoader extends ContainerAwareLoader
 {
+    private $loadedFixtures = array();
+
+    public function addFixtures(array $fixtures)
```

Comment:
Technically, this would require us to release this fix as 3.1.0 due to the new public method in a class. However, since the class is final and the method is there just for internal use in a bugfix, I'd be ok releasing this as 3.0.1 if we mark the method as `@internal`, thus telling people that they shouldn't use it. Could you add that?

Reply:
Fixed. Added `@internal`